omod/src/main/webapp/resources/scripts/services/privilegeService.js

```diff
@@ -0,0 +1,38 @@
+angular.module('privilegeService', ['ngResource', 'uicommons.common'])
+	.config(function ($httpProvider) {
+	})
+    .factory('Privilege', function($resource) {
+        return $resource("/" + OPENMRS_CONTEXT_PATH  + "/ws/rest/v1/privilege/:uuid", {
+            uuid: '@uuid'
+        },{
+            query: { method:'GET' }     // override query method to specify that it isn't an array that is returned
+        });
+    })
+    .factory('PrivilegeService', function(Privilege) {
+
+        return {
+
+            /**
+             * Fetches Privilege
+             *
+             * @param params to search against
+             * @returns $promise of array of matching Privileges (REST ref representation by default)
+             */
+            getPrivileges: function(params) {
```

Comment:
There is code aware of their existence because you just added them, so if you are not calling them anywhere then they are redundant. In your angular module [here](https://github.com/openmrs/openmrs-module-adminui/pull/20/files#diff-fb9756616bee3f8b577e7e07da278e63R213), you are calling the standard save method on the ngResource. In openmrs by convention delete results in retire or void where as purge is what results in the actual DB delete. So in REST to do a purge, you need to add the 'purge' request parameter when calling delete and if you want to retire you add the '!purge' request parameter when calling delete